### PR TITLE
Update Node and Yarn versions

### PR DESCRIPTION
**Pull request by lfdebrux**
We've update Node and Yarn in the Docker base images, so we should change the version requirements her accordingly.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,8 +9,8 @@ repo, run `make setup` and follow the instructions, then just run `make` to brin
 ## Requirements
 You must have the following tools available in order to successfully use the DM Runner:
 * Python 3 (including headers if appropriate) installed globally with `pip` and `virtualenv` packages.
-* Node v6.12.2 (consider using Node Version Manager), NPM 3+, and Yarn 1.3.2+ installed and available in your path.
-  * The command `nvm alias default 6.12.2` will alias the right version of Node for you
+* Node v8.12.0 (consider using Node Version Manager), NPM 6+, and Yarn 1.13.0+ installed and available in your path.
+  * The command `nvm use` will install and select the correct version of node for you
```
Comment (samuelhwilliams) [inline README.md]: 👍

Comment (samuelhwilliams) [reply README.md]: This doesn't work for me?

Comment (samuelhwilliams) [reply README.md]: screen shot 2019-01-24 at 10 42 22

Doesn't seem to install it for you?

Comment (lfdebrux) [reply README.md]: Thanks for testing, I have updated the suggested command to `nvm install && nvm use`

**Pull request merged**